### PR TITLE
style: change naming convention in order to stick to the contract

### PR DESCRIPTION
**Pull request by pik694**
clear distinction between utxo_pos and output_index

should be rather merged after: https://github.com/omisego/plasma-contracts/pull/59

### CODE REVIEW

File: apps/omg_eth/lib/eth/root_chain.ex [hunk 1]
```diff
@@ -116,12 +116,12 @@ defmodule OMG.Eth.RootChain do
     Eth.contract_transact(from_hex(from), contract, "addToken(address)", [token], opts)
   end
 
-  def challenge_exit(output_id, challenge_tx, input_index, challenge_tx_sig, from, contract \\ nil, opts \\ []) do
+  def challenge_exit(exit_id, challenge_tx, input_index, challenge_tx_sig, from, contract \\ nil, opts \\ []) do
```
[the remark this answers is not in the document]
Comment (pik694) [reply apps/omg_eth/lib/eth/root_chain.ex]: The motivation of these changes is to change names that were quite similar, but meant something totally different. 
What each means in the comment below.

**Comment by pik694** [comment]
`utxo_pos` - position of the UTXO (transaction _output_) - blknum, transaction index, index of the output
`txo_pos` -  position of the transaction - blknum, transaction index
`output_index` -  index of the output in a particular transaction
`exit_id` - an ID of either IFE or SE by which contract maps them, in order to get the ID , one should call contract function, which returns ID based on `utxo_pos` (SE) or tx_bytes (IFE).

File: apps/omg_eth/lib/eth/root_chain.ex [hunk 2]
```diff
@@ -173,11 +173,11 @@ defmodule OMG.Eth.RootChain do
...
 
     args = [
-      inflight_txbytes,
-      inflight_input_index,
+      in_flight_txbytes,
+      in_flight_input_index,
       competing_txbytes,
       competing_input_index,
-      competing_txid,
+      competing_tx_pos,
```
[the remark this answers is not in the document]
Comment (pik694) [reply apps/omg_eth/lib/eth/root_chain.ex]: `utxo_pos` is position of an output of a transaction while `tx_pos` is just a position of the transaction. For more see below.

File: apps/omg_eth/lib/eth/root_chain.ex [hunk 3]
```diff
@@ -236,7 +236,7 @@ defmodule OMG.Eth.RootChain do
   @doc """
   Returns exit for a specific utxo. Calls contract method.
   """
-  def get_exit(exit_id, contract \\ nil) do
+  def get_standard_exit(exit_id, contract \\ nil) do
```
Comment (pnowosie) [inline apps/omg_eth/lib/eth/root_chain.ex]: Regarding rename. How about [this one](https://github.com/omisego/elixir-omg/blob/fc0535bba9d09f817919969da03efc249de6c9f1/apps/omg_eth/lib/eth/root_chain.ex#L317), looks very similar, shouldn't be renamed as well?
Also, [this comment](https://github.com/omisego/elixir-omg/blob/fc0535bba9d09f817919969da03efc249de6c9f1/apps/omg_watcher/test/exit_processor/core_test.exs#L104) is missing a change.

File: apps/omg_watcher/lib/web/views/in_flight_exit.ex [hunk 4]
```diff
@@ -28,13 +28,13 @@ defmodule OMG.Watcher.Web.View.InFlightExit do
...
     competitor
-    |> Map.update!(:competing_txid, &Utxo.Position.encode/1)
+    |> Map.update!(:competing_tx_pos, &Utxo.Position.encode/1)
     |> OMG.RPC.Web.Response.serialize()
   end
 
   def render("prove_canonical.json", %{response: prove_canonical}) do
     prove_canonical
-    |> Map.update!(:inflight_txid, &Utxo.Position.encode/1)
+    |> Map.update!(:inflight_tx_pos, &Utxo.Position.encode/1)
```
Comment (pnowosie) [inline apps/omg_watcher/lib/web/views/in_flight_exit.ex]: We should pick one of `inflight` | `in_flight` and be consistent everywhere

File: apps/omg_eth/lib/eth/root_chain.ex (hunk 2 above)
[the remark this answers is not in the document]
Comment (pnowosie) [reply apps/omg_eth/lib/eth/root_chain.ex]: For `utxo_pos` we have `Utxo.Position` stuff. How to convert `tx_pos` to the contract expected form then?

> For more see below.

Can you link, where to look? Thx

File: docs/in_flight_exit_scenarios.md [hunk 5]
```diff
@@ -241,7 +241,7 @@ response:
         "competing_txbytes": "F317010180808080940000...",
         "competing_input_index": 1,
         "competing_sig": "9A23010180808080940000...",
-        "competing_txid": 2600003920012,
+        "competing_tx_pos": 2600003920012,
```
Comment (pnowosie) [inline docs/in_flight_exit_scenarios.md]: I'd like to know how to decode it to `{blknum, txindex}`

Comment (pik694) [reply docs/in_flight_exit_scenarios.md]: You should simply use `UTXO.Position`, but in this case `oindex` field is not important, though using `0` as its value is recommended.

File: apps/omg_watcher/priv/swagger/watcher_api_specs.yaml [hunk 6]
```diff
@@ -474,20 +474,20 @@ responses:
...
               data:
-                inflight_txbytes: '0xf3170101c0940000...'
-                inflight_input_index: 1
+                in_flight_txbytes: '0xf3170101c0940000...'
+                in_flight_input_index: 1
                 competing_txbytes: '0x5df13a6bee20000...'
                 competing_input_index: 1
                 competing_sig: '0xa3470101c0940000...'
-                competing_txid: 2600003920012
+                competing_tx_pos: 2600003920012
```
Comment (pnowosie) [inline apps/omg_watcher/priv/swagger/watcher_api_specs.yaml]: Following conversation [here](https://github.com/omisego/elixir-omg/pull/403#discussion_r252998294):
```
iex(3)> 2600003920012 |> Utxo.Position.decode()
{:utxo_position, 2600, 392, 12}
```
Above have 2 drawbacks:
1. Suggest impossible transaction in deposit block
1. Misleading 12th oindex

Can you please improve these accordingly? Thanks!

**Pull request merged**